haddock-api/src/Haddock/Interface.hs

```diff
@@ -165,6 +167,7 @@ createIfaces verbosity flags instIfaceMap mods = do
 processModule :: Verbosity -> ModSummary -> [Flag] -> IfaceMap -> InstIfaceMap -> Ghc (Maybe Interface)
 processModule verbosity modsum flags modMap instIfaceMap = do
   out verbosity verbose $ "Checking module " ++ moduleString (ms_mod modsum) ++ "..."
+  liftIO $ hSetEncoding stderr utf8
```

Comment:
> Also: Why would you want to bootstrap? An old version of cabal surely can compile cabal HEAD. So you don't need to take route via bootstrap.

Well, as long as I know, I think we need cabal-install built with GHC 8.2 to build cabal-install for GHC 8.2 (i.e. same version of GHC is required).
Then, currently I have no cabal-install built with GHC 8.2. That's why I'm trying to build cabal-install by the bootstrap script.
If I get something wrong, feel free to tell me!